Source/Resources/Content/MSGraph/DeviceManagement/DeviceConfigurations/OS - RDP Enabled.json

```diff
@@ -8,7 +8,7 @@
       }
     }
   ],
-  "description": "Enables RDP access to the device via CSP.",
+  "description": "What is it: Enables RDP access to the device for user that have remote access permissions.",
```

Comment:
**What does this do?**
Enables Remote Desktop access to the device for users that are members of the 'Remote Desktop Users' local group.

**Why should you use this?**
 If you want to allow Remote Desktop access to the device.

**What is the end-user impact?**
Users in the 'Remote Desktop Users' local group will be able to connect to remote devices. 

**Learn more**
[Tactical considerations for creating Windows deployment rings](https://techcommunity.microsoft.com/t5/windows-it-pro-blog/tactical-considerations-for-creating-windows-deployment-rings/ba-p/746979)